src/components/common/Select/Select.component.tsx

```diff
@@ -23,72 +23,64 @@ export const SelectPosition = {
+  ref: React.Ref<HTMLSelectElement>,
+) => {
+  const [isOpened, setOpened] = useState(false);
 
-    const toggleOpened = () => {
-      setOpened(!isOpened);
-    };
+  const [selectedOption, setSelectedOption] = useState(options[0]);
 
-    const handleClickOption = (optionValue: string) => {
-      setValue(optionValue);
-      toggleOpened();
-    };
+  const outerRef = useRef<HTMLDivElement>(null);
 
-    const selectedLabel = useMemo(
-      () => options.find((option) => option.value === value)?.label ?? '',
-      [options, value],
-    );
+  const toggleOpened = () => {
+    setOpened(!isOpened);
+  };
 
-    useOnClickOutSide(outerRef, () => setOpened(false));
+  const handleClickOption = (option: SelectOption) => {
+    setSelectedOption(option);
+    toggleOpened();
+  };
 
-    return (
-      <div ref={outerRef}>
-        <Styled.SelectContainer ref={ref}>
-          <Styled.Select size={size} onClick={toggleOpened} isOpened={isOpened} position={position}>
-            {value ? (
-              <Styled.SelectValue>{selectedLabel}</Styled.SelectValue>
-            ) : (
-              <Styled.SelectPlaceholder>{placeholder}</Styled.SelectPlaceholder>
-            )}
-            <ChevronDown />
-          </Styled.Select>
-          <Styled.SelectMenu isOpened={isOpened} position={position}>
-            {options.map((option) => (
-              <Styled.SelectOption
-                isSelected={value === option.value}
-                key={option.value}
-                onClick={() => handleClickOption(option.value)}
-              >
-                {option.label}
-              </Styled.SelectOption>
-            ))}
-          </Styled.SelectMenu>
-        </Styled.SelectContainer>
-      </div>
-    );
-  },
-);
+  useOnClickOutSide(outerRef, () => setOpened(false));
 
-export default Select;
+  return (
+    <div ref={outerRef}>
+      <Styled.SelectContainer>
+        <Styled.Select size={size} onClick={toggleOpened} isOpened={isOpened} position={position}>
+          {selectedOption ? (
+            <Styled.SelectValue>{selectedOption.label}</Styled.SelectValue>
+          ) : (
+            <Styled.SelectPlaceholder>{placeholder}</Styled.SelectPlaceholder>
+          )}
+          <ChevronDown />
+        </Styled.Select>
+        <Styled.SelectMenu isOpened={isOpened} position={position}>
+          {options.map((option) => (
+            <Styled.SelectOption
+              isSelected={selectedOption.value === option.value}
+              key={option.value}
+              onClick={() => handleClickOption(option)}
+            >
+              {option.label}
+            </Styled.SelectOption>
+          ))}
+        </Styled.SelectMenu>
+      </Styled.SelectContainer>
+      {/* SelectField를 위해 보이지 않는 select 추가 */}
+      <Styled.HiddenSelect ref={ref} value={selectedOption.value} disabled>
+        {options.map((option) => (
+          <option key={option.value} value={option.value}>
```

Comment:
```suggestion
          <option key={option.value}>
```

value property 는 필요 없어보이는데 맞을까요?

Reply:
Select는 우선 잠시 안쓰기로.. 하였습니다